src/config/types.ts

```diff
@@ -0,0 +1,37 @@
+export interface AppConfig {
+  mode: 'development' | 'production';
+  carbonApi: string;
+  appUrl: string;
+  walletConnectProjectId: string;
+  sentryDNS?: string;
+  rpcUrl: string;
+  addresses: {
+    tokens: {
+      ETH: string;
+      WETH: string;
+      BNT: string;
+      ZERO: string;
+      USDT: string;
+      USDC: string;
+      DAI: string;
+      WBTC: string;
+      SHIB: string;
+      ENJ: string;
+      UNI: string;
+      LINK: string;
+      LDO: string;
+      APE: string;
+      GRT: string;
+      AAVE: string;
+      CRV: string;
```

Comment:
If the idea of having a config is to facilitate app interactions with deployments on other chains, this type enforces that all these tokens exist on that chain, which most likely won't be the case

Reply:
I created an issue to discuss this: https://github.com/bancorprotocol/carbon-app/issues/1042